packages/flutter/test/material/text_field_test.dart

```diff
@@ -1443,6 +1443,33 @@ void main() {
     expect(controller.text, 'abc d${testValue}ef ghi');
   });
 
+  // Show the selection menu at the given index into the text by tapping to
+  // place the cursor and then tapping on the handle.
+  Future<void> _showSelectionMenuAt(WidgetTester tester, TextEditingController controller, int index) async {
+    await tester.tapAt(tester.getCenter(find.byType(EditableText)));
+    await tester.pump();
+    await tester.pump(const Duration(milliseconds: 200)); // skip past the frame where the opacity is zero
+    await tester.pumpAndSettle();
```

Comment:
Why do we need to pump a third time here?

Reply:
I tried it and it seems to be unnecessary, so I'll remove it.